spec/Conticket/Model/Aggregates/Event/TicketLifespanSpec.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace spec\Conticket\Model\Aggregates\Event;
+
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Conticket\Model\Aggregates\Event\TicketEndDateMustBeGreaterThanStartDateException;
+
+class TicketLifespanSpec extends ObjectBehavior
+{
+    function it_is_initializable()
```

Comment:
When using : void

Uncaught TypeError: Return value of spec\Conticket\Model\Aggregates\Event\TicketLifespanSpec::it_is_initializable() must be an instance of spec\Conticket\Model\Aggregates\Event\void, none returned in /Library/WebServer/Documents/conticket/spec/Conticket/Model/Aggregates/Event/TicketLifespanSpec.php:15